.flake8

```diff
@@ -0,0 +1,12 @@
+# A minimal "try to respect --line-length, but don’t become crazy if you can’t"
+# configuration as copied from black's documentation:
+# https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
+#
+[flake8]
+max-line-length = 88
+select = C,E,F,W,B,B950
```

Comment:
When using auto formatters, I think it's best to ignore C, E, W, since that's the linter checking the formatter. I think linters should only be used for correctness, and it's actually bad software practice to auto-enforce any style that cannot be applied by an auto formatter.